crates/viewer/re_data_ui/src/item_ui.rs

```diff
@@ -337,15 +338,17 @@ fn entity_tree_stats_ui(
         " (excluding subtree)"
     };
 
+    let store = db.store().read();
```

Comment:
Either that goes into the else block, or we need to modify all these subtree_stats method to explicitly take a store handle.

Either way we need a note regarding recursive locking.